packages/calcite-components/src/components/action-group/action-group.tsx

```diff
@@ -58,6 +58,11 @@ export class ActionGroup
     this.menuOpen = false;
   }
 
+  /**
+   * Specifies the label of the component.
+   */
+  @Prop() label!: string;
```

Comment:
This will force developers to add a label if they were using `action-group` already, which would be breaking. Can we document it as required and not enforce it by type until later? It is [publicly documented](https://developers.arcgis.com/calcite-design-system/components/action-group/).